cpp/ps0005/problem 5.cpp

```diff
@@ -10,42 +11,46 @@ int main() {
     cout << "Enter second number:\n";
     cin >> num2;

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input